src/seed_tools/utils/study_validation.ts

```diff
@@ -80,11 +88,97 @@ function checkExperiments(study: Study): string[] {
     }
     experimentNames.add(experiment.name);
     totalProbability += experiment.probability_weight ?? 0;
+
+    // Validate features.
+    if (experiment.feature_association !== undefined) {
+      if (
+        experiment.feature_association.enable_feature !== undefined &&
```

Comment:
Can we use `experiment?.feature_association` to remove ` if (experiment.feature_association !== undefined)`?
Also, `!==undefined` can't be get rid off if it's an object.